docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -287,6 +287,9 @@
           <para>Optionally you can install the package with rpm (recommended
           using the -Uvh options), however then you would have to negotiate
           installing any additional dependencies.</para>
+
+          <para>One way to install the dependencies on a CentOS system:
+          <programlisting lang="bash">sudo yum install -y epel-release</programlisting></para>
```

Comment:
The epel-release repository should be installed before the hpccsystems platform rpm package. We should introduce this para up around line 283 of the current file. At the end of the first paragraph of the CentOS/Red Hat installation instructions.